code/modules/vending/medical_wall.dm

```diff
@@ -11,12 +11,8 @@
 		/obj/item/reagent_containers/pill/charcoal = 1,
 		/obj/item/reagent_containers/medigel/styptic = 1,
 		/obj/item/reagent_containers/medigel/silver_sulf = 1,
-		/obj/item/reagent_containers/medigel/sterilizine = 1
-	)
-	contraband = list(
-		/obj/item/reagent_containers/pill/tox = 1,
```

Comment:
I think toxins would be fine to leave in the contraband, its one of the worse poisons.  HP+ Gum also seems random to remove?

Reply:
Want the gum in contraband or normal vendor?